lib/models/singleton-addon-cache/cache-entry.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+/**
+ * Class for the entries in the SingletonAddonCache. The cache itself is keyed
+ * on the addon's PackageInfo.root.
+ *
+ * @class CacheEntry
+ */
+class CacheEntry {
+  constructor(instance) {
```

Comment:
Instance of what? Addon instance?

Reply:
resolving as outdated.

Reply:
actually, ended up calling it cacheableAddonInstance and changing the internal name to _addonInstance.